packages/base/src/commonTest/kotlin/elide/util/Base64Test.kt

```diff
@@ -23,7 +23,7 @@ class Base64Test: AbstractEncoderTest<Base64>() {
       "should get expected output from non-websafe Base64 encode"
     )
     assertEquals(
-      "YWJjMTIzMTIzMQ",
+      "YWJjMTIzMTIzMQ==",
```

Comment:
this was wrong in the test. i was not aware that "urlsafe" base64 was different from "padless" base64. i am now.